lib/src/source/hosted.dart

```diff
@@ -419,10 +419,20 @@ class HostedSource extends CachedSource {
       if (retracted is! bool) {
         throw FormatException('retracted must be a bool');
       }
+      final advisoriesUpdated = body['advisoriesUpdated'];
+      if (advisoriesUpdated != null && advisoriesUpdated is! String) {
```

Comment:
Perhaps if you move the `is! String` check into the condition below, you can avoid the cast.

Reply:
done